core/app_modules/email.js

```diff
@@ -0,0 +1,38 @@
+var nodemailer = require('nodemailer');
+var auth = require('./auth')
+var utils = require('./utils')
+
+const ENV = process.env;
+
+
+var transporter = nodemailer.createTransport({
+  service : ENV.EMAILSERVICE,
+  auth:{
+    user: ENV.EMAIL,
+    pass: ENV.EMAILPASS
+  }
+});
+
+function Reservation(from, reserveID){
+  var mailOptions = {
+      from: from.attributes.name + '<' + from.attributes.emailAddress + '>', // sender address
+      to: ENV.EMAIL, // list of receivers
+      subject: 'Room Reservation for ' + from.attributes.name, // Subject line
+      text: 'VEIW THE HTML n00blet.', // plaintext body
+      html: '<head><style> div>a{text-decoration: none; height: 25px; display: block; width: 100%; font-size: 25px; text-align: center; border-radius: 10px; color: black;}\
```

Comment:
Possibly. Every functions should have one in that case though. Then be included.